geopackage_validator/validations/rtree_valid_check.py

```diff
@@ -19,15 +20,22 @@ def rtree_valid_check_query(dataset) -> Iterable[str]:
         "and extension_name = 'gpkg_rtree_index');"
     )
     for index in indexes:
-        validations = dataset.ExecuteSQL(
-            'select rtreecheck("{index_name}");'.format(
-                index_name="rtree_" + index[0] + "_" + index[1]
+        with silence_gdal():
+            validations = dataset.ExecuteSQL(
+                'select rtreecheck("{index_name}");'.format(
+                    index_name="rtree_" + index[0] + "_" + index[1]
+                )
             )
-        )
-        for validation in validations:
-            if validation[0] != "ok":
-                yield validation[0]
-        dataset.ReleaseResultSet(validations)
+
+            if validations is None:
+                yield index[0]
+                dataset.ReleaseResultSet(validations)
+                continue
+
+            for validation in validations:
```

Comment:
Het maakt niet zoveel uit. Maar als ik het zo bezie kan je waarschijnlijk ermee wegkomen om deze for loop te silencen. Dat is het moment wanneer die error gegooid wordt.

Reply:
Hmm. Net even getest door "with silence_gdal():" boven de loop "for validation in validations:" te plaatsen, maar dan komt de gdal error weer naar boven.